share/spice/twitter/skipwords.txt

```diff
@@ -1,7 +1,21 @@
+about
+analytics?
 apis?
+app(s)?
```

Comment:
seeing as only the `s` is optional here, we don't need to use a capturing group for any of these:

```perl
apps?
basics?
bots?
...
```

the `?` in these cases applies only to the `s` allowing for `app` or `apps`. the `(es)?` is necessary because we're group "es" and making that sub-string optional, without the group we'd need `e?s?`